Automattic/wp-calypso
# Jetpack Cloud: Add Monetize section

Slack: p1711036278716469/1711029747.007349-slack-CKZHG0QCR

## Proposed Changes

We add `Monetize` section  to Jetpack Cloud.

## Known issues
- `Learn more` on Monetize subtitle links to WPcom support, Doc requested here: https://github.com/Automattic/jpop-quill/issues/158
- Privacy `Learn more` & `Privacy Information` links to WPcom support. Docs requested here: https://github.com/Automattic/jpop-quill/issues/167
- `Automattic Ads Terms of Service`  links to WPcom support. Docs requested here: https://github.com/Automattic/jpop-quill/issues/167
- Launchpad links to the wrong path, PR created (https://github.com/Automattic/jetpack/pull/36728) and should be deployed after we release Monetize on JP Cloud

## Tasks

- [x] Define icon for `Monetize` > Updated to `payment`
- [x] Should we maintain `/earn` or replace with `/monetize`? > Changed to `/monetize`
- [x] Check CSS. > CSS Updated.

### Tabs

#### Monetization Options

Atomic sites:
- [x] `Collect payments` & `Receive donations and tips` links to `https://wordpress.com/payments-donations/`. Maybe replace with `https://jetpack.com/support/jetpack-blocks/donations-block/`
- [x] `Create subscriber-only content` links to `https://wordpress.com/support/wordpress-editor/blocks/premium-content-block/`. Replaced it to `https://jetpack.com/support/jetpack-blocks/paid-content-block/`
- [x] `Set up a paid newsletter` links to `https://wordpress.com/support/paid-newsletters/`. Replaced with `https://jetpack.com/support/newsletter/paid-newsletters/`
- [x] `Collect PayPal payments` links to `https://wordpress.com/support/wordpress-editor/blocks/pay-with-paypal/`. Replaced with `https://jetpack.com/support/pay-with-paypal/`
- [x] `Refer a friend` has WPcom related copy and refer link is a WPcom link, removed from Self Hosted. 

Self-hosted sites:
- [x] `Collect payments`, `Receive donations and tips`, and `Collect PayPal payments`. Same as Atomic
- [x] `Earn ad revenue` with plan links to `/earn/ads-settings`. No plan links to `/pricing/` and adds `Learn more` that links to `wordads.co`

#### Supporters
- [x] `Learn more` links to `https://wordpress.com/support/wordpress-editor/blocks/payments/`. Replaced with `https://jetpack.com/support/jetpack-blocks/payments-block/`

#### Payment Settings
- [x] `Connect Stripe` opens an Stripe window but WPcom branded and back link redirects to WPcom. We can rebrand in a follow-up

## Testing Instructions

> [!NOTE]  
> To test the email design when joining WordAds, you have to apply this diff: D144101-code

* Run `yarn start-jetpack-cloud `
* Go to `http://jetpack.cloud.localhost:3000/monetize/:your-site`
* Check the copies.
* Check if support/help links to Jetpack support docs at Jetpack Cloud and WPcom docs on WPcom.
* Check if everything is working.
* Do it with an Atomic site and a self-hosted site.
* Check if nothing is broken on Calypso at `/earn/:your-site`
